schema.graphql

```diff
@@ -318,12 +318,11 @@ type Mutation {
   updateProfileFields(input: [UpdateProfileInput]!): Person
   uploadProfileImage(file: Upload!, size: Int!): Person
   addPrayer(text: String!, isAnonymous: Boolean): Prayer
-  interactWithPrayer(id: ID!, action: PrayerAction!): Prayer
-  deletePrayer(nodeId: String!): Prayer @deprecated(reason: "Use interactWithPrayer(action:DELETE)")
-  incrementPrayerCount(nodeId: String!): Prayer @deprecated(reason: "Use interactWithPrayer(action:INCREMENT)")
-  flagPrayer(nodeId: String!): Prayer @deprecated(reason: "Use interactWithPrayer(action:FLAG)")
-  savePrayer(nodeId: String!): Prayer @deprecated(reason: "Use interactWithPrayer(action:SAVE)")
-  unSavePrayer(nodeId: String!): Prayer @deprecated(reason: "Use interactWithPrayer(action:UNSAVE)")
```

Comment:
@redreceipt I'm pretty sure this file shouldn't be in here. I don't think you want to remove these changes.